soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -4824,16 +4825,9 @@ void CreateGetItemMessages(std::vector<GetItemMessage> messageEntries) {
     CustomMessageManager* customMessageManager = CustomMessageManager::Instance;
     customMessageManager->AddCustomMessageTable(Randomizer::getItemMessageTableID);
     for (GetItemMessage messageEntry : messageEntries) {
-        if (messageEntry.giid == RG_ICE_TRAP) {
-            customMessageManager->CreateMessage(Randomizer::getItemMessageTableID, messageEntry.giid,
-                                                { TEXTBOX_TYPE_BLUE, TEXTBOX_POS_BOTTOM, messageEntry.english,
-                                                  messageEntry.german, messageEntry.french });
-        } else {
-            customMessageManager->CreateGetItemMessage(Randomizer::getItemMessageTableID, messageEntry.giid,
-                                                       messageEntry.iid,
-                                                       { TEXTBOX_TYPE_BLUE, TEXTBOX_POS_BOTTOM, messageEntry.english,
-                                                         messageEntry.german, messageEntry.french });
-        }
+        customMessageManager->CreateGetItemMessage(Randomizer::getItemMessageTableID, messageEntry.giid, messageEntry.iid,
+                                                    { TEXTBOX_TYPE_BLUE, TEXTBOX_POS_BOTTOM, messageEntry.english,
+                                                        messageEntry.german, messageEntry.french });
```

Comment:
just want to verify that this call works when the `giid` is `RG_ICE_TRAP` (we have the `iid` for the `messageEntry` etc.)

Reply:
I'm not quite sure what you mean. The ice trap messages bypass the regular getItemMessages completely now, is that what you're talking about?

Reply:
so the `messageEntries` in this vector will never have any messages with `RG_ICE_TRAP` as the `giid`? perfect!

Reply:
Yeah, the old ice trap message entry is removed lower down in this same file in the `Randomizer::CreateCustomMessages()` function.